apps/server/src/utils/parser.ts

```diff
@@ -195,11 +195,15 @@ export const parseExcel = (excelData: unknown[][], options?: Partial<ImportMap>)
       const column = row[j];
       // 1. we check if we have set a flag for a known field
       if (j === timerTypeIndex) {
-        if (column === 'block') {
+        if (typeof column !== 'string' || column !== undefined) {
+          return;
+        }
+        const maybeTimeType = typeof column === 'string' ? column.trim() : '';
```

Comment:
question: should this be something we do to all string imports? I cant imagine a scenario where we would want whitespace in a string

Reply:
that sounds reasonable